### ISSUE TITLE
use onnx model to build engine error

### ISSUE THREAD

**Issue by lllllinux** [keddyjin/TensorRT_StableDiffusion_ControlNet#1]
## Description
 i use the demo_txt2img.py in demo/Diffusion,it works when using default arguments. but when i use the dynamic shape ,it fails with the error below:

[E] 10: Could not find any implementation for node {ForeignNode[onnx::LayerNormalization_9097 + (Unnamed Layer* 1230) [Shuffle].../down_blocks.0/attentions.0/Reshape_1 + /down_blocks.0/attentions.0/Transpose_1]}.
[E] 10: [optimizer.cpp::computeCosts::3869] Error Code 10: Internal Error (Could not find any implementation for node {ForeignNode[onnx::LayerNormalization_9097 + (Unnamed Layer* 1230) [Shuffle].../down_blocks.0/attentions.0/Reshape_1 + /down_blocks.0/attentions.0/Transpose_1]}.)
[!] Invalid Engine. Please ensure the engine was built correctly

## Environment

**TensorRT Version**: 8.6

**NVIDIA GPU**: rtx4090

**NVIDIA Driver Version**: 12.0

**CUDA Version**: 11.6

**CUDNN Version**:8.4.0

Operating System: ubuntu 20.04

Python Version (if applicable): 3.10

Tensorflow Version (if applicable): none

PyTorch Version (if applicable): 1.13.1

Baremetal or Container (if so, version):

**Comment by lllllinux** [comment]
i just build the engine succeessfully with a100.and with minimizing the max image resolution, i can also build engine file in rtx 4090. But the same problem in both gpu is that the engine file with dynamic shape built will cost more time than static shape in inference.

**Comment by keddyjin** [comment]
Sure, couldn't be more normal. The tactics used for dynamic shape engine are not the optimal compared with static shape.
How much slower is the dynamic shape version?

**Comment by lllllinux** [comment]
> Sure, couldn't be more normal. The tactics used for dynamic shape engine are not the optimal compared with static shape. How much slower is the dynamic shape version?

actually, the inference preformance is really weird.
when i use this repo, the dynamic shape feature will slower the inference process. 50steps, 512 resolution costs about 2.5 seconds.
but when i use the diffusers repo to convert model to onnx and use trtexec to get .trt file, the inference process was not effected by dynamic shape feature. but this way will cause solid black output image. i'm still working on it.
and besides, i'm also working on loading lora on unet engine file dynamically with refit feature. but i'm stuck in modify onnx weights according to state_dict saved in torch. cuz weights name and shape will change when the model was converted to onnx from torch. i can't match them. do you have some advices?

**Comment by keddyjin** [comment]
I haven't tested the onnx->trtexec pipeline. I you can export the model to onnx successfully , it should also work well. You may firstly need to check if the onnx model can produce the correct image.

For lora model, you should add lora weights to the original unet model by mapping the weights name directly. You can do this before exporting to onnx. This operation will not affect the weights name or shape. Then you can use TensorRT refit feature to change the weights without rebuilding.
